stick_to_it/main_app/models.py

```diff
@@ -1,3 +1,8 @@
 from django.db import models
 
 # Create your models here.
+class Card(models.Model):
+    activity = models.CharField(max_length=100)
+    due_date = models.DateField
+    complete_date = models.DateField
+    
```

Comment:
can you remove this extra line? @mlsmoorman

Reply:
Nope...it's hard coded... :) @bradsm1th